service/switchboardapp/contracts/api-spec.yaml

```diff
@@ -0,0 +1,234 @@
+openapi: 3.0.0
+info:
+  version: 1.0.0
+  title: 'Switchboard API'
+  description: 'API for the switchboard service'
+
+servers:
+  - url: https://placeholder.com/v1/
+    description: Test environment
+
+paths:
+  /Encoders:
+    get:
+      operationId: getEncoders
+      description: Get a list of available encoders
+      tags:
+        - Encoders
+      responses:
+        '200':
+          description: Successful retrieval of encoder list
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/EncoderUuids'
+    post:
+      operationId: createEncoder
+      description: Create a new encoder client
+      tags:
+        - Encoders
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/CreateEncoderRequest'
+      responses:
+        '200':
+          description: Successful creation the encoder
+  /Encoders/{uuid}:
```

Comment:
Have we decided on using universally unique identifier (uuid)? Why not just an id?

Reply:
That's pretty much just what I've been doing from experience. Another idea was to use the database IDs, but I figured that would be error prone. Do you have any other ideas?